### ISSUE TITLE
[Card Present Payments] Epic Phase 1

### ISSUE THREAD

**Issue by malinajirka** [woocommerce/woocommerce-android#3724]
This is a parent task for Phase 1 of the "Card Present Payments" project on Android. This project contains 4 major subtasks:
- [x] Connecting a BT reader to a phone (issue https://github.com/woocommerce/woocommerce-android/issues/3725)
- [x] Collecting a payment from a buyer (i.e. in-person payment) (issue https://github.com/woocommerce/woocommerce-android/issues/3726)
- [x] Allowing the buyer to get an immediate hardcopy receipt (i.e. printing receipt) or if they prefer one over email (issue https://github.com/woocommerce/woocommerce-android/issues/4039)
- [x] Updating the card reader firmware (issue https://github.com/woocommerce/woocommerce-android/issues/3957)
- [x] Make sure we use WooLog (not AppLog) everywhere
- [x] ⚠️  Change LogWrapper (lives in CardReaderModule) to an interface and implement it in the app using WooLog (https://github.com/woocommerce/woocommerce-android/pull/4319)
- [x] Move card reader module classes to release package (PR https://github.com/woocommerce/woocommerce-android/pull/4299)
- [x] Enable the feature flag (PR https://github.com/woocommerce/woocommerce-android/pull/4299)
- [x] !!!!**Make sure StripeTerminalLogs are disabled in production**!!!!
- [x] Update CI config so tests in CardReaderModule use "releaseTest" instead of "debugTest" (https://github.com/woocommerce/woocommerce-android/pull/4299)
- [x] Add "Getting started" documentation (PR https://github.com/woocommerce/woocommerce-android/pull/4516)

**Comment by wzieba** [comment]
I'll link comment about using `AppLog` vs. `WooLog` https://github.com/woocommerce/woocommerce-android/pull/3969#discussion_r630900065 as it might be valuable for `Make sure we use WooLog (not AppLog) everywhere` point.

TLDR; I'd advice to indeed use `WooLog` with custom tag but for other reasons than originally highlighted.
